### PR TITLE
Feature/lvpytcdxii 16

### PR DESCRIPTION
**Pull request by LSikyr**
* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=1032)

## Code reviewers

- [ ] @sikyrynskiy
- [ ] @olya_petryshyn
- [ ] @taraskonchak
- [ ] @OlyaKh00
- [ ] @ement06
- [ ] @iPavliv
- [ ] @khadai
- [ ] @Anastasia_Siromska
- [ ] @romichh

### Second Level Review

- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,4 +6,4 @@ install:
   - pip install -r requirements.txt
 
 script:
-  - pylint answers_service run.py
\ No newline at end of file
+  - pylint answers_service setup.py
```
Comment (iPavliv) [inline .travis.yml:9]: add new line after

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+prod-env:
+	 make install; \
+	 export PYTHONPATH=$(PYTHON_PATH_ANSWERS_SERVICE); \
+	 export FLASK_APP="setup.py"; \
+	 export FLASK_ENV="production"; \
+	 flask run --port=5000;
+
+lint:
+	pylint setup.py answers_service/
```
Comment (iPavliv) [inline Makefile:44]: add new line after

File: answers_service/migrations/env.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+import logging
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+# this is the Alembic Config object, which provides
```
Comment (iPavliv) [inline answers_service/migrations/env.py]: delete comments

File: answers_service/migrations/versions/8a1d754f422c_.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    sa.Column('group_id', sa.Integer(), nullable=False),
+    sa.PrimaryKeyConstraint('id')
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('answer')
+    # ### end Alembic commands ###
```
Comment (iPavliv) [inline answers_service/migrations/versions/8a1d754f422c_.py]: delete comments

File: answers_service/views/answers.py [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+        # check if such answer exists
+        exists = bool(Answer.query.filter_by(user_id=user_id, form_id=form_id,
+                                             group_id=group_id, field_id=field_id).first())
+        if exists:
+            result = ({'error': 'this answer alreasy exist'}, 203)
+        else:
+            new_answer = Answer(reply=reply, user_id=user_id, form_id=form_id,
+                                field_id=field_id, group_id=group_id)
+            DB.session.add(new_answer)  # pylint: disable=no-member
```
Comment (iPavliv) [inline answers_service/views/answers.py:55]: add to .pylintrc to generated members: add, commit, delete, query, rollback

**Review by ement06 — APPROVED** [review]

**Pull request merged**